src/components/SearchAndFilter/SearchAndFilter.test.js

```diff
@@ -22,4 +22,18 @@ describe("Search and filter ", () => {
     wrapper.find(".p-search-box__input").simulate("change");
     expect(mockOnChange).toHaveBeenCalledTimes(1);
   });
+
+  it("shows panel on focus", () => {
```

Comment:
Can you add a test for the click or is that difficult?

Reply:
I don't think we need to test that the browser adds focus on click?

Reply:
Gotcha.. added